pagopa/io-app
# feat: [IOCOM-497] PN cancelled message UI

## Short description
This PR adds support for PN cancelled messages at the UI level.

![Screenshot 2023-09-12 at 15 36 47](https://github.com/pagopa/io-app/assets/5150343/b69a2260-2115-4345-959d-662ca0b80ba6)

## List of changes proposed in this pull request
- On a cancelled message, a banner is displayed
- On a cancelled message, the attachment list appearance is different and the user interaction is disabled

## How to test
Make sure to have the local feature flag on for the new PN component. Run the io-dev-api-server with a cancelled PN message configuration and check that:
- the banner is displayed
- attachment list is not fully opaque
- attachment tap does nothing

Also check that all of the above are not valid for a standard PN message (i.e., not cancelled)
